strawberryfields/io.py

```diff
@@ -80,10 +80,15 @@ def to_blackbird(prog, version="1.0"):
                     op["kwargs"]["dark_counts"] = cmd.op.dark_counts
 
         else:
+            isMeasuredParameter = lambda x: isinstance(x, sfpar.MeasuredParameter)
```

Comment:
(Minor): maybe worth moving this out of the for loop

Reply:
Oh, haha, @albi3ro actually commented on the same thing, but I only moved it outside the first for-loop. 😆